### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by Samb092** [tailwindlabs/tailwindcss#7690] [Help]
arsenetshibanda@Arsenes-Mini Prince Ndaba % npm run dev
npm ERR! Missing script: "dev"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/arsenetshibanda/.npm/_logs/2022-02-27T08_42_46_751Z-debug-0.log

HOW DO I GET TO FIX THIS ERROR?

**Comment by asmsaiff** [comment]
Can you show me your package.json file ?? Maybe you don't create any dev command to run that on the terminal.

**Reply by Samb092** [reply]
{
  "name": "prince-ndaba",
  "version": "1.0.0",
  "main": "script.js",
  "devDependencies": {
    "autoprefixer": "^10.4.2",
    "postcss": "^8.4.7",
    "tailwindcss": "^3.0.23",
    "vite": "^2.8.4"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "description": ""
}

**Reply by asmsaiff** [reply]
Yah. You didn't create the dev command inside this scripts - 

```json
"scripts": {
"test": "echo "Error: no test specified" && exit 1"
},
```

Follow this command to install tailwind CSS without any error - 

- ```npm init -y```
- ```npm install -D tailwindcss```
- ```npx tailwindcss init```
- Paste following code into ```tailwind.config.js``` inside your project folder
```javascript
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
- Create tailwind CSS input file named ```input.css``` and paste the following code inside of this file -
```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```
- Open your ```package.json``` in a code editor
- Create ```npx tailwindcss -i ./<file_path>/input.css -o ./<file_path>/style.css --watch``` this command in scripts object like this -
```json
"scripts": {
"test": "echo "Error: no test specified" && exit 1",
"dev": "npx tailwindcss -i ./<file_path>/input.css -o ./<file_path>/style.css --watch"
},
```
- Run ```npm run dev``` in your terminal
- Enjoy!

It should work perfectly.

**Reply by Samb092** [reply]
Thank you so much let me try it out, if i get stuck i'll let you know

**Reply by asmsaiff** [reply]
You're welcome. Let me know if it works or not. We'll figure out the problems.

**Reply by asmsaiff** [reply]
@Samb092 Have your issue fixed yet or not??

**Comment by quickeasy818** [comment]
Hi, can anyone help me with this?

npm ERR! Missing script: "dev"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Local\npm-cache\_logs\2023-02-24T11_29_02_240Z-debug-0.log
